app/assets/stylesheets/members/index.css.scss

```diff
@@ -53,6 +95,19 @@ $parallax-background-height: $parallax-window-height;
   width: 100%;
 }
 
+@include breakpoint($mobile) {
+  .parallax-background {
+    background: url("parallax.jpg");
+    background-size: cover;
```

Comment:
Properties should be ordered background, background-color, background-position, background-size, left, position, top, width